rewrite-gradle/src/test/java/org/openrewrite/gradle/UpgradeDependencyVersionTest.java

```diff
@@ -435,4 +436,29 @@ void defaultsToLatestRelease() {
           )
         );
     }
+
+    @Test
+    void versionInPropertiesFile() {
```

Comment:
I think we should also add a test case that models a multi module project layout where the version is found within a child's `gradle.properties` file.